src/cascadia/TerminalApp/AppLogic.idl

```diff
@@ -58,7 +60,8 @@ namespace TerminalApp
         event Windows.Foundation.TypedEventHandler<Object, String> TitleChanged;
         event Windows.Foundation.TypedEventHandler<Object, LastTabClosedEventArgs> LastTabClosed;
         event Windows.Foundation.TypedEventHandler<Object, Windows.UI.Xaml.ElementTheme> RequestedThemeChanged;
-        event Windows.Foundation.TypedEventHandler<Object, ToggleFullscreenEventArgs> ToggleFullscreen;
         event Windows.Foundation.TypedEventHandler<Object, ToggleFocusModeEventArgs> ToggleFocusMode;
+        event Windows.Foundation.TypedEventHandler<Object, ToggleFullscreenEventArgs> ToggleFullscreen;
+        event Windows.Foundation.TypedEventHandler<Object, AlwaysOnTopChangedEventArgs> AlwaysOnTopChanged;

```

Comment:
ghghgh consistency

so we're going from an event that says "flip your state" (with the two toggles) to an event that says "state changed, ask me for the new one"? should we align one or the other of the categories?

Reply:
Eh, yes. This is now weird between the three. The "the state changed, come ask what it is" is more idiomatic, and flexible, so I'll update the others too.